src/components/ratings.js

```diff
@@ -1,44 +1,48 @@
 import React, { Component } from 'react';
-import {connect} from 'react-redux';
-
+import { connect } from 'react-redux';
 import Ratings from 'react-ratings-declarative';
 
 import { updateBeer } from '../actions/index';
 
 export var ratingChange = false;
```

Comment:
what is it for? Aslo not var, but `let`

Reply:
this was for an earlier version of the code, I've removed it